modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/log/LogBuffer.kt

```diff
@@ -110,6 +111,14 @@ object LogBuffer {
         addLog(Log.INFO, tag, null, message, *args)
     }
 
+    fun i(tag: String, throwable: Throwable, message: String, vararg args: Any) {
+        addLog(Log.INFO, tag, throwable, message, *args)
+    }
+
+    fun log(priority: Int, tag: String, throwable: Throwable, message: String, vararg args: Any) {
+        addLog(priority, tag, throwable, message, *args)
+    }
```

Comment:
Instead of adding a  new `log` method, could we get the same effect by renaming `addLog` to `log` and making that public? Certainly not a big deal, but it feels a bit more straightforward.